Firestore/Example/Tests/SpecTests/json/listen_spec_test.json

```diff
@@ -3858,6 +3856,25 @@
           },
           "targetId": 2
         },
+        "expectedSnapshotEvents": [
+          {
+            "errorCode": 0,
+            "fromCache": true,
+            "hasPendingWrites": false,
+            "query": {
+              "filters": [
+                [
+                  "visible",
+                  "==",
+                  true
+                ]
+              ],
+              "orderBys": [
+              ],
+              "path": "collection"
+            }
+          }
+        ],
```

Comment:
I trust that these json files were generated correctly :)

Reply:
Lol! You _can_ verify by running the export yourself ;)